docs/devices/E1812.md

```diff
@@ -36,8 +36,13 @@ The [binding](../guide/usage/binding.md) functionallity of this remote varies pe
 - below 2.3.75: suppports binding to groups only. It can only be bound to 1 group at a time. By default this remote is bound to the default bind group which you first have to unbind it from. This can be done by sending to `zigbee2mqtt/bridge/request/device/unbind` payload `{"from": "DEVICE_FRIENDLY_NAME", "to": "default_bind_group"}`. Wake up the device right before sending the commands by pressing a button on it.
 - 2.3.075 and greater: supports binding to devices only
 
-### Troubleshooting: no battery level
+## Troubleshooting
+
+### No battery level
 It may help to remove the battery for a few seconds and after that reconfigure it via [Configure](../guide/usage/mqtt_topics_and_messages.md#zigbee2mqttbridgeconfigure). Right before executing the Configure make sure to wake up the device by pressing a button on it.
+
+### No communication after pairing
+As proposed by [Heisenberg2980](https://github.com/Heisenberg2980) in [this comment](https://github.com/Koenkk/zigbee2mqtt/issues/9439#issuecomment-956456214) the workaround to this issue is to bind it to the coordinator after pairing by sending the mqtt message `zigbee2mqtt/bridge/request/device/bind` with payload `{"from": "<device friendly name>", "to": "coordinator"}`
```

Comment:
Can you check if just binding `genOnOff` and `genLevelCtrl` is enough? (you can do this via the z2m frontend -> device -> bind). If yes I can do this out-of-the box which prevents users for having to do this manually.

Reply:
Sorry. I had a child on oct 27 and this got pushed way down my priority queue. I will open or re-commit should this issue persist once I get around to fiddling with zigbee2mqtt again... if ever.

Thanks :)